app/com/linkedin/drelephant/tez/heuristics/ReducerMemoryHeuristic.java

```diff
@@ -30,18 +30,21 @@
 
   private static final Logger logger = Logger.getLogger(ReducerMemoryHeuristic.class);
 
-  public static final String REDUCER_MEMORY_CONF = "mapreduce.reduce.memory.mb";
+  public static final String MAPRED_REDUCER_MEMORY_CONF = "mapreduce.reduce.memory.mb";
+  public static final String HIVE_MAPPER_MEMORY_CONF = "hive.tez.container.size";
+  public static final String TEZ_MAPPER_MEMORY_CONF = "tez.task.resource.memory.mb";
```

Comment:
I think TEZ_MAPPER_MEMORY_CONF and HIVE_MAPPER_MEMORY_CONF can be defined in GenericMemoryHeuristic so they don't need to be defined in both MapperMemoryHeuristic and MapperMemoryHeuristic.